### ISSUE TITLE
[Bug] Game crush when switching to Windows mode-v1.9.5.2072

### ISSUE THREAD

**Issue by gceboh** [bjaraujo/Bombermaaan#29]
Firstly,thank you for your hard work!

I want to report a bug found in the latest release.

**Release Version:** 1.9.5.2072
**Platform:** Windows 8.1 x64

**Description:**
1.Everytime I press F4 in order to switch to the Windows mode, the program will always immediately crash.
2.No CrashReports found in `%AppData%\..\Local\CrashRpt\UnsentCrashReports`.

Note that **No bug found in the previous version**:1.9.4.2045.

I guess that this bug may be caused either by the [the update of the visual studio](https://github.com/bjaraujo/Bombermaaan/commit/8528b59c83cdba05274f5e016a4810cc17d29d54) ,or the new introduced [ogg sound system](https://github.com/bjaraujo/Bombermaaan/commit/6726ceb2af85b460c92eaf43903517fec54e3dfb).But I'm not sure.

**Comment by bjaraujo** [comment]
It seems to crash in SDL so I think it is an SDL mixer bug. I have marked the latest release as pre-release so feel free to use the previous release.

**Comment by gceboh** [comment]
Thank you for reply, I know this bug may be hard to fix.

**Comment by gceboh** [comment]
A workaround for switching to windows mode is to modify `config.xml`.

change this line: `<DisplayMode value="3" />`
to: `<DisplayMode value="4" />`

---

The previous version(1.9.4.2045) doesn't have this bug,which has the same SDL version.
I've compared each file between current version and the previous version(1.9.4.2045),and found that only these files has been changed:
```
Bombermaaan.exe
Bombermaaan.pdb
Bombermaaan32.dll
config.xml
CrashRpt1403.dll
CrashSender1403.exe
msvcp140.dll
vcruntime140.dll
```

So bug must lie in one of these files. A quite strange bug.@bjaraujo
